cache/nydus.go

```diff
@@ -0,0 +1,164 @@
+//go:build nydus
+// +build nydus
+
+package cache
+
+import (
+	"compress/gzip"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/errdefs"
+	"github.com/moby/buildkit/cache/config"
+	"github.com/moby/buildkit/session"
+	"github.com/moby/buildkit/solver"
+	"github.com/moby/buildkit/util/compression"
+	digest "github.com/opencontainers/go-digest"
+	ocispecs "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+
+	nydusify "github.com/containerd/nydus-snapshotter/pkg/converter"
+)
+
+var validAnnotations = append(
+	compression.EStargzAnnotations, containerdUncompressed,
+	nydusify.LayerAnnotationNydusBlob, nydusify.LayerAnnotationNydusBootstrap, nydusify.LayerAnnotationNydusBlobIDs,
+)
```

Comment:
Should be `additionalAnnotations` and `compression.EStargzAnnotations, containerdUncompressed,` should be in refs.go and remote.go.

Reply:
Fixed.